packages/lodestar/src/util/chunkify.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Split an inclusive range into a sequence of contiguous inclusive ranges
+ * ```
+ * [[a,b], [c,d] ... Sn] = chunkifyInclusiveRange([a,z], n)
+ * // where
+ * [a,z] = [a,b] U [c,d] U ... U Sn
+ * ```
+ * @param from range start inclusive
+ * @param to range end inclusive
+ * @param chunks Maximum number of chunks, if range is big enough
+ */
+export function chunkifyInclusiveRange(from: number, to: number, chunkCount: number): number[][] {
```

Comment:
isn't sync chunkify doing same thing?

Reply:
Right, didn't know that function existed. What do you suggest to do, to move sync chunkify to utils and make it more generic?

Reply:
Hmm yeah, let's do that

Reply:
Can we do that in another PR not minimize the diff?

Reply:
hmm shouldn't be too much diff, it had identical interface, right?

Reply:
No, the interface is different. The `chunkifyInclusiveRange` of this PR passes as parameter `chunkCount` while `chunkify` of sync passes `itemsPerChunk`